addons/website_slides_forum/views/website_slides_forum_templates.xml

```diff
@@ -26,4 +26,89 @@
             </t>
         </xpath>
     </template>
+
+    <template name="Split regular forums/courses" id="website_slides_forum_index" inherit_id="website_forum.forum_all" active="True" customize_show="True">
+        <xpath expr="//div[@id='o_wforum_forums_index_list']" position="replace">
+            <div id="o_wforum_forums_index_list" class="container pt-4 pb-5">
+                <t t-if="forums">
+                    <!-- Check if only one 'regular' forum exist (beside courses) -->
+                    <t t-set="single_regular_forum" t-value="len(forums) - len(forums.slide_channel_id) == 1"/>
```

Comment:
Seems wtf and buggy...

Reply:
@qsm-odoo , have you read this ? https://github.com/odoo/odoo/pull/35749#issuecomment-533856641